operators/derive/__init__.py

```diff
@@ -0,0 +1,12 @@
+from . import to_dp, to_es, to_mapbox, to_sql
+
+if __name__ == '__main__':
```

Comment:
we need an `operator` function here for dgp to be able to run this.

Reply:
got it